packages/terra-form-input/CHANGELOG.md

```diff
@@ -3,6 +3,19 @@ ChangeLog
 
 Unreleased
 ----------
+### Breaking Changes
+* `placeholder` prop has been deprecated.
```

Comment:
We might want to change the verbs being used here. I think the ecosystem associates `deprecated` with still available but not recommended to be used, whereas here it's being removed entirely.

I'd recommend using `removed` instead of `deprecated` throughout the documentation

Reply:
Updated here https://github.com/cerner/terra-core/pull/3075/commits/f132725755a408088dd0b53919a344c9dadff568.